docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,104 @@
+module.exports = {
+  title: "SPAship",
+  tagline: "SPAship is an open source platform for deploying, integrating, and managing single-page apps (SPAs).",
+  url: "https://spaship.com",
+  baseUrl: "/",
+  favicon: "img/favicon.ico",
+  organizationName: "Red Hat", // Usually your GitHub org/user name.
```

Comment:
@kunyan The Git Org name for SPAship is SPAship.
```suggestion
  organizationName: "SPAship", // Usually your GitHub org/user name.
```